main.tf

```diff
@@ -91,163 +98,107 @@ resource "helm_release" "gitlab-runner-linux" {
+  wait       = false
+  namespace  = "sensitive"
+  # repository = "https://charts.gitlab.io/gitlab"
+  chart      = "https://gitlab-charts.s3.amazonaws.com/gitlab-runner-0.33.1.tgz"
   
   values     = [
-    file("${path.module}/gitlab-runner/win-values.yaml")
+    file("${path.module}/gitlab-runner/kaniko-values.yaml")
     ]
 
   set {
     name  = "gitlabUrl"
-    value = "${var.gitlab_instance_protocol}://${local.instance_internal_domain}"
+    value =  "${var.gitlab_instance_protocol}://${local.instance_internal_domain}"
   }
   set {
-    name  = "cloneUrl"
-    value = "${var.gitlab_instance_protocol}://${local.instance_internal_domain}"
+    name  = "runners.cloneUrl"
+    value =  "${var.gitlab_instance_protocol}://${local.instance_internal_domain}"
   }
   set {
     name  = "certsSecretName"
     value = "${local.instance_internal_domain}-cert"
   }
   set_sensitive {
     name  = "runnerRegistrationToken"
-    value = random_password.gitlab_runner_registration_token.result
+    value = "${random_password.gitlab_runner_registration_token.result}-scallops-recipes"
   }
 }
 
-########################### GKE Cluster #################################################
-
+resource "helm_release" "gitlab-runner-dockerhub" {
+  count  = var.dockerhub-creds-secret != "" ? 1 : 0  
+  depends_on = [
+                module.gke,
+                module.gke_auth,
+                kubernetes_namespace.sensitive-namespace
+                ]
+  name       = "dockerhub-privates"
+  wait       = false
+  namespace  = "sensitive"
+  chart      = "https://gitlab-charts.s3.amazonaws.com/gitlab-runner-0.33.1.tgz"
+  
+  values     = [
+    file("${path.module}/gitlab-runner/dockerhub-values.yaml")
+    ]
 
-resource "kubernetes_secret" "k8s_gitlab_cert_secret" {
-  depends_on  = [module.gke_auth, module.gke]
-  data        = {
-    "${local.instance_internal_domain}.crt" = tls_self_signed_cert.gitlab-self-signed-cert.cert_pem
+  set {
+    name  = "gitlabUrl"
+    value =  "${var.gitlab_instance_protocol}://${local.instance_internal_domain}"
   }
-  metadata {
-    name      = "${local.instance_internal_domain}-cert"
-    namespace = "default"
+  set {
+    name  = "runners.cloneUrl"
+    value =  "${var.gitlab_instance_protocol}://${local.instance_internal_domain}"
   }
-}
-
-
-resource "kubernetes_secret" "google-application-credentials" {
-  depends_on  = [module.gke_auth, module.gke]
-  data        = {
-    "kaniko-token-secret.json" = base64decode(google_service_account_key.storage_admin_role.private_key)
+  set {
+    name  = "certsSecretName"
+    value = "${local.instance_internal_domain}-cert"
+  }
+  set {
+    name  = "runners.imagePullSecrets[0]"
+    value = kubernetes_secret.dockerhub-creds-config[0].metadata[0].name
   }
-  metadata {
-    name      = "kaniko-secret"
-    namespace = "default"
+  set_sensitive {
+    name  = "runnerRegistrationToken"
+    value = random_password.gitlab_runner_registration_token.result
   }
```

Comment:
indentation.

Reply:
It is.
Can you take a look from here?
https://github.com/SygniaLabs/ScallOps/blob/v_0.3-update/main.tf#L159